index.html

```diff
@@ -93,6 +94,12 @@
 				maxime mollitia necessitatibus nobis pariatur quae quaerat quo. Doloremque, dolores eos necessitatibus tempora
 				tenetur vero.
 			</ts-note>
+			<article>
+				<h2>Textarea</h2>
+				<!--				<textarea name="text area" id="test" cols="30" rows="10"></textarea>-->
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a space at column 80. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=Web%3AWhiteSpaceAroundCheck)